### DISCUSSION TITLE
Prettier Plugin

### DISCUSSION THREAD

**Discussion by nderscore** [nativewind/nativewind#355] [Show and tell]
I've just published an initial version of a prettier plugin for automatically sorting NativeWind class names here:

> [`prettier-plugin-nativewind`](https://www.npmjs.com/package/prettier-plugin-nativewind)

It's a very light fork of the official `prettier-plugin-tailwindcss` plugin, featuring support for:

* Sorting tailwind classes in `className` and `tw` props from NativeWind.

* Sorting tailwind classes in `styled()` calls, including v3.x variants API.

If you run into any issues, please submit an [issue on my repo](https://github.com/nderscore/prettier-plugin-nativewind/issues) or, even better, a pull request 😎

**Comment by marklawlor** [comment]
Hey this is awesome! I'm currently rewriting the docs and I'll ensure to include this in a community section.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
